apps/vr-tests/src/stories/react-charting/LineChart.stories.tsx

```diff
@@ -0,0 +1,967 @@
+import * as React from 'react';
+import { Steps, StoryWright } from 'storywright';
+import { storiesOf } from '@storybook/react';
+import { TestWrapperDecorator } from '../../utilities/index';
+import { IChartProps } from '../../../../../packages/react-charting/src/HorizontalBarChart';
```

Comment:
why using relative paths and not @fluentui/react-charting.
Similarly for other imports also.